examples/pymc3_howto/custom_distribution.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>I would suggest adding a prior predictive step, not only because it's part of the standard Bayesian worflow, but also because it uses your distribution in a different way than in <code>sample</code> or <code>posterior_predictive</code>, as an unobserved random variable. That in itself of a good sanity check that the distribution is well implemented and can be sampled from (with mcmc that is)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I'm trying to add this, but I think it won't work because [pm.AR](https://docs.pymc.io/api/distributions/timeseries.html#pymc3.distributions.timeseries.AR) doesn't have a `random` method. Is it ok if I leave this step out?

Reply:
Actually what I had in mind was prior predictive sampling with `pm.sample` by simply removing the `observed` keyword. 

But that's not necessarily useful for this example...

Feel free to ignore it then

Reply:
I like your idea of doing a sanity check though! I think it would be nice to compare what samples look like from the standard Poisson vs generalized Poisson

Reply:
<p>That's a good point!</p>